docs/Manual/08 Deferred Layout of Constant Arrays.md

```diff
@@ -2,20 +2,20 @@
 [//]: # (Version: v1.2.1)
 
 # Section 8: Deferred layout of constant arrays
-We revisit the topic of memory layout of constant arrays. As mentioned in [Section 1](<01%20Arrays.md>), the contents of constant arrays are known at compile time, they are immutable, and they are not externally visible. This allows Accera to store them using a non-standard layout, optimized for a specific plan. In some cases, there may even be a benefit to storing multiple copies of each array element (e.g., storing a matrix in both row-major and column-major layouts).
+Let's revisit the memory layout of constant arrays. As explained in [Section 1](<01%20Arrays.md>), the contents of constant arrays are known at compile-time, and these contents are immutable and hidden externally. Accera stores constant arrays in a non-standard memory layout optimized for a particular plan. In some cases, storing multiple copies of each array element may even prove advantageous (e.g., storing a matrix in row-major and column-major layouts).
```

Comment:
we can remove "and hidden externally" (and basically also remove "not externally visible" from the original text), it doesn't really add much to what the user needs to know about.